README.md

```diff
@@ -183,6 +183,19 @@ make format
 make lint
 ```
 
+### Debugging
+
+Developer builds do not include debug information by default. If you want to attach a
+debugger to your local Relay, add a custom profile to your local `$HOME/.cargo/config.toml`:
+
+```toml
+[profile.mydebug]
+inherits = "dev"
+debug = true
```

Comment:
I'm now wondering if we should add this profile to the workspace Cargo.toml, might be useful for everyone.